subsystem_controllers/src/drivers_controller/ssc_driver_manager.cpp

```diff
@@ -49,7 +49,9 @@ carma_msgs::msg::SystemAlert SSCDriverManager::get_latest_system_alert(
     alert.type = carma_msgs::msg::SystemAlert::NOT_READY;
     return alert;
   } else if (!ssc_is_operational) {
-    alert.description = "SSC Failed";
+    // Driver takeover currently throws a SSC failure alert.
+    // This is a temporary workaround until the driver takeover alert is updated to be more descriptive
+    alert.description = "System shutting down";
```

Comment:
Maybe let's say SSC Shut down instead of System because people without context why this change happened still may need more clue. We just wanted to easen the language from failure to shut down per client request because this message is common for our normal operation until we add the specific case to detect the takeover